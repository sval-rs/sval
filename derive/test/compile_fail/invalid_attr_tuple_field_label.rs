#[macro_use]
extern crate sval_derive;

#[derive(Value)]
pub struct Record (
    #[sval(label = "foo")]
    i32,
    i32,
);

fn main() {}
