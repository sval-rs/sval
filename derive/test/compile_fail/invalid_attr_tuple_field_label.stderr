error: proc-macro derive panicked
 --> compile_fail/invalid_attr_tuple_field_label.rs:4:10
  |
4 | #[derive(Value)]
  |          ^^^^^
  |
  = help: message: unsupported attribute `label` on unnamed field
